client/connector_create.go

```diff
@@ -113,6 +114,17 @@ func (cli *VanClient) ConnectorCreateSecretFromData(ctx context.Context, options
 				return nil, fmt.Errorf("Claims not supported. %s", err)
 			}
 		}
+		if expirationStr, ok := secret.ObjectMeta.Annotations[types.ClaimExpiration]; ok {
+			expiration, timeErr := time.Parse(time.RFC3339, expirationStr)
+			if timeErr != nil {
+				err = timeErr
+				return nil, fmt.Errorf("timer error %s", err)
```

Comment:
I'd suggest we just print a warning here also, e.g. 'cannot parse expiration'

Reply:
fixed

Reply:
Its still returning an error rather than printing a warning.

Reply:
sorry,  just focused on what was being printed, vs type of error.   I have now fixed to just log a warning but to continue on processing.